nilearn/signal.py

```diff
@@ -429,6 +466,10 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
         raise TypeError("confounds keyword has an unhandled type: %s"
                         % confounds.__class__)
 
+    if normalize not in ("psc", "std", None):
+        raise ValueError("invalid value for 'normalize'"
+                         " parameter: {0}".format(normalize))
+
```

Comment:
You check this both in `_normalize` and here. For code maintenance, it would be better to do it in one location.